dot/sync/tip_syncer_integeration_test.go

```diff
@@ -207,18 +207,26 @@ func TestTipSyncer_handleTick_case2(t *testing.T) {
 			targetNumber: uintPtr(header.Number),
 			direction:    network.Ascending,
 			requestData:  network.RequestedDataBody + network.RequestedDataJustification,
-			pendingBlock: s.pendingBlocks.getBlock(header.Hash()),
+			pendingBlock: &pendingBlock{
+				hash:    header.Hash(),
+				number:  201,
+				header:  header,
+				clearAt: time.Time{},
+			},
 		},
 	}
 	w, err := s.handleTick()
 	require.NoError(t, err)
+	w[0].pendingBlock.clearAt = time.Time{}
```

Comment:
we should assert the `clearAt` field somehow (even if it's not an exact assertion, at least verify it's set in the future or something)